packages/fxa-auth-server/lib/payments/paypal/error-codes.ts

```diff
@@ -12,41 +12,10 @@ export const PAYPAL_RETRY_ERRORS = [10014, 10445, 11453, 11612];
  * funding source.
  */
 export const PAYPAL_SOURCE_ERRORS = [
-  10069,
-  10203,
-  10204,
-  10205,
-  10207,
-  10210,
-  10212,
-  10216,
-  10417,
-  10502,
-  10504,
-  10507,
-  10525,
-  10527,
-  10537,
-  10546,
-  10554,
-  10555,
-  10556,
-  10560,
-  10567,
-  10600,
-  10601,
-  10606,
-  10606,
-  10748,
-  10752,
-  11084,
-  11091,
-  11458,
-  11611,
-  13109,
-  13122,
-  15012,
-  18014,
+  10069, 10203, 10204, 10205, 10207, 10210, 10212, 10216, 10417, 10502, 10504,
+  10507, 10525, 10527, 10537, 10546, 10554, 10555, 10556, 10560, 10567, 10600,
+  10601, 10606, 10606, 10748, 10752, 11084, 11091, 11458, 11611, 13109, 13122,
+  15012, 18014,
```

Comment:
No idea why prettier decided this was the new cool way to do it....

Reply:
Also, 2022 and Github still don't have a syntax aware diff viewer.